pymc/sampling/mcmc.py

```diff
@@ -237,6 +237,8 @@ def _sample_external_nuts(
     model: Model,
     progressbar: bool,
     idata_kwargs: Optional[Dict],
+    nuts_kwargs: Optional[Dict],
+    sampler_kwargs: Optional[Dict],
```

Comment:
What about calling it `nuts_sampler_kwargs`, in line with `nuts_sampler` to choose the sampler?

Reply:
Good idea